src/objectivec/oga_internal.mm

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "oga_internal.h"
+
+@implementation OGAInt32Span {
+  const int32_t* _ptr;
+  size_t _size;
+}
+
+- (nullable)initWithRawPointer:(const int32_t*)pointer size:(size_t)size {
+  _ptr = pointer;
+  _size = size;
+  return [self init];
+}
+
+- (const int32_t*)pointer {
+  return _ptr;
+}
+
+- (size_t)size {
+  return _size;
+}
+
+- (int32_t)last {
+  return *(_ptr + (_size - 1));
+}
```

Comment:
What's the use case for this? std::span has things like `begin` and `end` but that's slightly different to returning the value of the last element (and this would also do the wrong thing if size was 0 wouldn't it)?